sway-fmt/src/traversal_helper.rs

```diff
@@ -278,4 +402,36 @@ mod tests {
             "a::{bar, foo};"
         );
     }
+
+    #[test]
+    fn test_format_use_statement_length_leaves_input_unchanged() {
+        let s = "a::b::{c, d::{self, f}};";
+        assert_eq!(format_use_statement_length(s, 100, 0), s);
+    }
+
+    #[test]
+    fn test_format_use_statement_length_formats_long_input() {
+        let s = "std::{address::*, assert::assert, block::*, chain::auth::*, context::{*,text::{call_frames::*, dial_frames::{Transaction, TransactionParameters}, token_storage::{CallData, Parameters}}}, contract_id::ContractId, hash::*, panic::panic, storage::*, token::*};";
+        let expected = "---std::{
+---    address::*,
+---    assert::assert,
+---    block::*,
+---    chain::auth::*,
+---    context::{
+---        *,
+---        text::{
+---            call_frames::*,
+---            dial_frames::{Transaction,TransactionParameters},
+---                token_storage::{CallData,Parameters,
```

Comment:
1. Why is there an extra indentation here?
2. Can you use the `ALREADY_FORMATTED_LINE_PATTERN` constant here? I know it'll be kind of cumbersome, but the alternative is hard-coding the literal 😂